packages/node/Dockerfile

```diff
@@ -1,14 +1,24 @@
-# production images
+# Build stage
 FROM node:18 as builder
-ARG RELEASE_VERSION
-ENTRYPOINT ["subql-node"]
-RUN npm i -g --unsafe-perm @subql/node@${RELEASE_VERSION}
 
+WORKDIR /app
+COPY ./packages/node .
+
+RUN npm install -g --force yarn@latest
+RUN yarn install
+
+
+# Production stage
 FROM node:18-alpine
 ENV TZ utc
 
 RUN apk add --no-cache tini git curl
-COPY --from=builder /usr/local/lib/node_modules /usr/local/lib/node_modules
 
-ENTRYPOINT ["/sbin/tini", "--", "/usr/local/lib/node_modules/@subql/node/bin/run"]
-CMD ["-f","/app"]
+COPY --from=builder /app/package.json ./
+COPY --from=builder /app/yarn.lock ./
+COPY --from=builder /app/node_modules ./node_modules
+COPY --from=builder /app/dist ./dist
+COPY --from=builder /app/bin ./bin
```

Comment:
Ideally we can use `.npmignore` to determine what to copy. (we might need to make some updates to `.npmignore`)

Reply:
modified the dockerfile to pack the node package with yarn before copying to docker, this way we only get files that will be in the npm package